hail/src/main/resources/include/hail/NDArray.h

```diff
@@ -0,0 +1,72 @@
+#ifndef HAIL_NDARRAY_H
+#define HAIL_NDARRAY_H 1
+
+#include <vector>
+
+struct NDArray {
+  int flags;
```

Comment:
Would be great to document somewhere nearby what the `flags` are here. From what I can gather, it's currently just the least significant bit is `row_major`?

Reply:
Good point, and yes that's currently the only thing it's saying.